Assets/Scripts/SS3D/Systems/IngameConsoleSystem/Commands/HurtCommand.cs

```diff
@@ -13,50 +13,27 @@ namespace SS3D.Systems.IngameConsoleSystem.Commands
 {
     public class HurtCommand : Command
     {
-        public override string LongDescription => "Hurt a given body part and body layer of a player by a given amount of damages from a given type";
+        public override string LongDescription => "Hurt body layer in a body part of a player";
+        public override string Usage => "(game object name) (BodyLayerType) (DamageType) (amount of damage)";
```

Comment:
change usage to (Player ckey)(Body part name) (body layer name) (damage type) (damage amount)
you can also add an exemple like hurt EditorUser HumanHead all toxic 10